index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Frederick's-Portfolio</title>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <link rel="stylesheet" href="styles.css">
+
+</head>
+<body>
+       <nav class="nav-bar">
+        <h1 class="Logo">My Logo</h1>
+        <img src="Images/Union.png" alt="Three-Lines">
+       </nav>
```

Comment:
- [ ] Please The container of the toolbar section must be a `header` tag

- [ ] Please the  hamburger button must be on the right 
![image](https://user-images.githubusercontent.com/74744787/176787036-8866aff7-34ab-4fde-a216-e8316710c90e.png)